contracts/staking/compound/GoodCompoundStaking.sol

```diff
@@ -11,8 +11,8 @@ import "../UniswapV2SwapHelper.sol";
  * or withdraw their stake in Token
  * the contracts buy cToken and can transfer the daily interest to the  DAO
  */
-contract GoodCompoundStaking is SimpleStaking {
-	using UniswapV2SwapHelper for SimpleStaking;
```

Comment:
this should not be changed. SimpleStaking should implement IHasRouter

Reply:
changed